DonjinKrawler-Client/src/main/java/donjinkrawler/Game.java

```diff
@@ -5,6 +5,7 @@
 import donjinkrawler.decorator.MaracasEnemy;
 import donjinkrawler.decorator.PonchosEnemy;
 import donjinkrawler.decorator.SombrerosEnemy;
+import donjinkrawler.facade.MusicMaker;
```

Comment:
Unused import. You should really check the checkstyle warnings before you commit. 
Since this applies to all of us, I'll clean up once this is merged, no need to do anything.